### PR TITLE
ENH: updated for qiime/q2-feature-classifier#62

### PR DESCRIPTION
**Pull request by BenKaehler**
Won't work until qiime/q2-feature-classifier#62 is merged.

### CODE REVIEW

File: source/tutorials/feature-classifier.rst [hunk 1]
```diff
@@ -81,7 +81,7 @@ Finally, we verify that the classifier works by classifying the representative s
 
 .. command-block::
 
-   qiime feature-classifier classify \
+   qiime feature-classifier classify-sklearn \
```
Comment (jairideout) [inline source/tutorials/feature-classifier.rst]: Can you also make this fix in the Moving Pictures tutorial? I think that's the only other place needing updates.

Comment (BenKaehler) [inline source/tutorials/feature-classifier.rst]: Thanks @jairideout. Fixed.

**Comment by jairideout** [comment]
Thanks @BenKaehler!

**Comment by BenKaehler** [comment]
Thanks @jairideout, and @gregcaporaso.

**Pull request merged**